app/controllers/offerings_controller.rb

```diff
@@ -0,0 +1,30 @@
+class OfferingsController < ApplicationController
+
+  #show all offers; we don't use this but page exists
+  def index
+    @offers = Offering.all
+  end
+
+  def new
+    @offer = Offering.new
+    @boats = Boat.all
```

Comment:
Dit was om de dropdown met alle boten te generen in het 'create new offer'
form

Misschien is het niet nodig? Dat moet ik, na alle nieuwe info van gisteren,
checken.


Met vriendelijke groet,
Kind regards,

Iris Bune

On Wed, Feb 10, 2016 at 10:37 AM, navase <notifications@github.com> wrote:

> In app/controllers/offerings_controller.rb
> <https://github.com/navase/boatconnection/pull/14#discussion_r52434096>:
>
> > @@ -0,0 +1,30 @@
> > +class OfferingsController < ApplicationController
> > +
> > +  #show all offers; we don't use this but page exists
> > +  def index
> > +    @offers = Offering.all
> > +  end
> > +
> > +  def new
> > +    @offer = Offering.new
> > +    @boats = Boat.all
>
> Wat doet @boats <https://github.com/boats> = Boat.all hier? Bij een
> nieuwe offer een overzicht geven van alle boten?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/navase/boatconnection/pull/14/files#r52434096>.
>